node/pkg/watchers/evm/watcher.go

```diff
@@ -124,6 +127,9 @@ type (
 	}
 )
 
+// DefMaxConfs is default value for the maximum number of confirmations to wait before declaring a transaction abandoned.
+const DefMaxConfs = 32
```

Comment:
can we just spell out default? because I missed this multiple times on reading through.

Reply:
Got rid of this constant.